app/models/validations/household_validations.rb

```diff
@@ -161,53 +161,53 @@ def validate_previous_housing_situation(record)
     return unless record.owning_organisation
 
     if record.is_internal_transfer? && record.owning_organisation.provider_type == "PRP" && record.is_prevten_la_general_needs?
-      record.errors.add :prevten, :internal_transfer_fixed_or_lifetime, message: I18n.t("validations.household.prevten.la_general_needs.internal_transfer")
-      record.errors.add :referral, :internal_transfer_fixed_or_lifetime, message: I18n.t("validations.household.referral.la_general_needs.internal_transfer")
+      record.errors.add :prevten, :internal_transfer_fixed_or_lifetime, message: I18n.t("validations.lettings.household.prevten.la_general_needs.internal_transfer")
+      record.errors.add :referral, :internal_transfer_fixed_or_lifetime, message: I18n.t("validations.lettings.household.referral.la_general_needs.internal_transfer")
     end
   end
 
   def validate_prevloc(record)
     if record.previous_la_known? && record.prevloc.blank?
-      record.errors.add :prevloc, I18n.t("validations.household.previous_la_known")
+      record.errors.add :prevloc, I18n.t("validations.lettings.household.prevloc.previous_la_known")
     end
   end
 
   def validate_layear(record)
     return unless record.layear && record.renewal
 
     if record.is_renewal? && record.layear == 1
-      record.errors.add :layear, :renewal_just_moved, message: I18n.t("validations.household.renewal_just_moved_to_area.layear")
-      record.errors.add :renewal, I18n.t("validations.household.renewal_just_moved_to_area.renewal")
+      record.errors.add :layear, :renewal_just_moved, message: I18n.t("validations.lettings.household.layear.renewal_just_moved_to_area")
+      record.errors.add :renewal, I18n.t("validations.lettings.household.renewal.renewal_just_moved_to_area")
     end
   end
 
   def validate_layear_and_prevloc(record)
     return unless record.layear && record.la && record.prevloc && record.collection_start_year
 
     if record.la == record.prevloc && record.layear == 1 && record.collection_start_year >= 2023
-      record.errors.add :layear, :renewal_just_moved, message: I18n.t("validations.household.same_la_just_moved_to_area.layear")
-      record.errors.add :la, :renewal_just_moved, message: I18n.t("validations.household.same_la_just_moved_to_area.current_la")
-      record.errors.add :postcode_full, :renewal_just_moved, message: I18n.t("validations.household.same_la_just_moved_to_area.current_la")
-      record.errors.add :uprn, :renewal_just_moved, message: I18n.t("validations.household.same_la_just_moved_to_area.current_la")
-      record.errors.add :ppostcode_full, :renewal_just_moved, message: I18n.t("validations.household.same_la_just_moved_to_area.previous_la")
-      record.errors.add :prevloc, :renewal_just_moved, message: I18n.t("validations.household.same_la_just_moved_to_area.previous_la")
+      record.errors.add :layear, :renewal_just_moved, message: I18n.t("validations.lettings.household.layear.same_la_just_moved_to_area.current_la")
```

Comment:
It looks like the `.current_la` or `.previous_la` at the end of each of these might be performing a slightly more generic version of what we're doing with putting the field in (i.e. allowing different messages depending on where you're seeing them). Do we actually need both, or could these all be just e.g. `validations.letttings.household.layear.same_la_just_moved_to_area`?

Reply:
Would that also be for the rest too (`la`, `uprn`, `prevloc`)? Only `postcode_full` has both `current_la` and `previous_la`. `ppostcode_full` also only has `previous_la`

Reply:
Does `postcode_full` actually use both? Presumably it would be in this list if it did, and there's only current used here (also it would be surprising I think to have a previous la type thing attached to `postcode_full` rather than `ppostcode_full`).

I think this applies to all 6 fields/error messages in this validation, unless `postcode_full` is using a previous la version somewhere I can't see.

Reply:
Ah yes it doesn't actually use the previous_la, don't why I included it. Will remove it and put these all directly under `same_la_just_moved_to_area`

Reply:
Updated